log2ram.conf

```diff
@@ -29,5 +29,5 @@ COMP_ALG=lz4
 # LOG_DISK_SIZE is expected compression ratio of alg chosen multiplied by log SIZE
 # lzo/lz4=2.1:1 compression ratio zlib=2.7:1 zstandard=2.9:1
 # Really a guestimate of a bit bigger than compression ratio whilst minimising 0.1% mem usage of disk size
-LOG_DISK_SIZE=100M
+LOG_DISK_SIZE=60M
```

Comment:
Please don't change the default of this file.
You can add it on gitignore to avoid this problem

Reply:
Look once more I am actually doing the work and showing you.
If I change so when working it saves me having to change it on each test I will do so.
Just change it back.
In fact was still working on it as have done things in steps so you can see and it is back but whilst I am working with it I will change it